pkg/streamingpromql/operators/aggregation.go

```diff
@@ -101,12 +110,8 @@ type group struct {
 	// Used to sort groups in the order that they'll be completed in.
 	lastSeriesIndex int
 
-	// Sum, presence, and histograms for each step.
-	floatSums              []float64
-	floatCompensatingMeans []float64 // Mean, or "compensating value" for Kahan summation.
-	floatPresent           []bool
-	histogramSums          []*histogram.FloatHistogram
-	histogramPointCount    int
+	// The AggregationGroup to perform over this group of series.
```

Comment:
[nit] Not sure this comment is adding much, but if you want to keep it, perhaps "the aggregation for this group of series" would be clearer